charts/piraeus/templates/external-metrics-service.yaml

```diff
@@ -0,0 +1,39 @@
+{{ if (.Values.externalController).url }}
+---
+{{- $url := urlParse .Values.externalController.url -}}
+{{- $host := regexSplit ":" (get $url "host") -1 -}}
+{{- $ipAddr := index $host 0 -}}
+{{- $port := index $host 1 }}
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: {{ include "piraeus-operator.fullname" . }}-external-controller-metrics-service
+  labels:
+  {{- include "piraeus-operator.labels" . | nindent 4 }}
+    app.kubernetes.io/component: linstor-external-controller
+subsets:
+  - addresses:
+    - ip: {{ $ipAddr }}
+    ports:
+    - name: metrics
+      port: {{ $port }}
+      protocol: TCP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "piraeus-operator.fullname" . }}-external-controller-metrics-service
```

Comment:
This name also needs to be set on the `ServiceMonitor` resource instead of the usual controller metrics.

Reply:
This is not really clear to me. My patch has no ServiceMonitor. It is installed from config/extras.

Reply:
Should I simply remove "_external_" from *external-controller* names?